mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/session/impl/StandalonePojoSearchSession.java

```diff
@@ -237,7 +287,55 @@ public SearchSessionBuilder loading(Consumer<SelectionLoadingOptionsStep> loadin
 
 		@Override
 		public StandalonePojoSearchSession build() {
+			boolean lowLevelConfig = ( refreshStrategy != null || commitStrategy != null );
+			if ( synchronizationStrategy != null && lowLevelConfig ) {
+				throw log.conflictingIndexPlanSynchronizationConfiguration();
+			}
+			else if ( synchronizationStrategy == null && lowLevelConfig ) {
+				synchronizationStrategy = strategy(
+						Optional.ofNullable( commitStrategy ).orElse( DocumentCommitStrategy.FORCE ),
+						Optional.ofNullable( refreshStrategy ).orElse( DocumentRefreshStrategy.NONE )
+				);
+			}
+			else if ( synchronizationStrategy == null ) {
+				synchronizationStrategy = IndexingPlanSynchronizationStrategy.writeSync();
+			}
```

Comment:
Don't bother, the standalone Pojo mapper is a new module which was only introduced in 6.2.0.Alpha1. We can break APIs as we please. In fact I wouldn't even document the API break.

Just remove support for setting `refreshStrategy`/`commitStrategy` directly?